--- gqrx-2.3.0/gqrx.pro~	2014-08-05 21:14:41.000000000 +0200
+++ gqrx-2.3.0/gqrx.pro	2015-06-19 12:30:17.734906712 +0200
@@ -27,14 +27,6 @@ macx {
 # enable pkg-config to find dependencies
 CONFIG += link_pkgconfig
 
-unix:!macx {
-    packagesExist(libpulse libpulse-simple) {
-        # Comment out to use gr-audio (not recommended with ALSA and Funcube Dongle Pro)
-        AUDIO_BACKEND = pulse
-        message("Gqrx configured with pulseaudio backend")
-    }
-}
-
 RESOURCES += icons.qrc \
     textfiles.qrc
 
